CHANGELOG.md

```diff
@@ -6,13 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.9.0] - 2021-05-04
+## [0.9.0] - 2021-05-05
+### Added
+- Configure `GOPATH` and `TF_DATA_DIR` and add both as docker volumes
 ### Changed
 - Upgrade Terraform to v0.15.1
 - Upgrade TfLint to v0.28.0
 - Upgrade pre-commit to v2.12.1
 - Upgrade snyk to v1.556.0
 
+## [0.8.1] - 2021-05-05
+### Added
+- Configure `GOPATH` and `TF_DATA_DIR` and add both as docker volumes
+
```

Comment:
this is already a breaking change ;) don't think we need to release a 0.8.1

Reply:
resolved in 5dcff2f